src/modelldcatnotordf/agent.py

```diff
@@ -0,0 +1,72 @@
+"""Agent module for mapping a model to rdf.
+
+This module contains methods for mapping a agent object to rdf
+for use in the modelldcat-ap-no specification._
+
+Refer to sub-class for typical usage examples.
+"""
+from typing import Optional
+
+from datacatalogtordf import URI
+from rdflib import BNode, Graph, Literal, Namespace, RDF, URIRef
+
+FOAF = Namespace("http://xmlns.com/foaf/0.1/")
+
+
+class Agent(BNode):
```

Comment:
Det er mulig vi bør legge denne ein annan stad. Men på kort sikt er det fint å implementere foaf:Agent her. #

Reply:
Flott Stig, takk! Synes også at det burde lages et bibliotek for foaf. Brukes jo mange steder. Men greit å kunne gjøre dette senere og så refaktorerer vi dette etterpå.. :+1: